solidity/contracts/TestERC721A.sol

```diff
@@ -0,0 +1,23 @@
+//SPDX-License-Identifier: Apache-2.0
+pragma solidity 0.8.10;
+import "@openzeppelin/contracts/token/ERC721/ERC721.sol";
+
+// One of three testing coins
```

Comment:
this comment needs to be updated, testing NFT or somthing.

Reply:
updated comment